xclim/indicators/atmos/_precip.py

```diff
@@ -619,3 +620,31 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.cold_and_wet_days,
 )
+
+rain_season = PrecipWithIndexing(
+    title="Rain season",
+    identifier="rain_season",
+    realm="atmos",
+    var_name=["start", "end", "length"],
+    standard_name=["rain_season_start", "rain_season_end", "rain_season_length"],
+    long_name=[
+        "Start of the rain season",
+        "End of the rain season",
+        "Length of the rain season",
+    ],
+    description=[
+        "First step of a run where i) a sequence of {window_wet_start} days accumulated {thresh_wet_start}"
+        "of precipitations ii) followed by a sequence of {window_not_dry_start} days with no dry sequence, i.e. a sequence of {window_dry_start} days"
+        "with at least {thresh_dry_start} {method_dry_start}. The start of the season is on the last day of the first sequence i) and must be between {date_min_start} and {date_max_start}",
+        "Last day in a dry sequence after the start of the season, i.e.  a sequence of {window_dry_end} days"
+        "with at least {thresh_dry_end} {method_dry_end}. It must be between {date_min_end} and {date_max_end}",
+        "Number of steps of the original series in the season, between 'start' and 'end'.",
+    ],
+    units="",
+    abstract="The rain season begins when two conditions are met: 1) There must be a number of wet days with"
+    "precipitations above or equal to a given threshold; 2) There must be another sequence following, where, for a given period in time, there are no"
+    "dry sequence (i.e. a certain number of days where precipitations are below or equal to a certain threshold). The rain season ends"
+    "when there is a dry sequence.",
```

Comment:
```suggestion
```
Better put it in the compute function so the latter is better documented.

Reply:
Should I simply remove the current `Notes` with this?